### PR TITLE
Workspace switching

### PR DESCRIPTION
**Pull request by pvh**
Here's a first implementation of workspace switching -- to test it, copy the URL from one client and paste it in another. You might want to save the workspace URLs on a couple of boards, first.

Next steps include:
 - designing support for remembering multiple workspaces
 - adding device presence per the sync-state branch
 - color-coding titlebar / omnibox header with presence color
 - improve workspace-in-list rendering
 - omnibox functional component... maybe.

If I have a bit of time tomorrow I'll work on 2 and 3 in this list (as separate patches) but I'd appreciate a review &| merge of this work so far.

## Changes
 * contact editor moved to full-screen
 * contact editor removed from top-right of title bar
 * contact list removed from top-right of title bar
 * current workspace depicted as avatar + briefcase icon in omnibox
 * workspace loading if you paste in a workspace and press enter

### CODE REVIEW

File: src/renderer/components/content-types/workspace/Device.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.DeviceListItem {
+  display: flex;
+  align-items: center;
+  width: 100%;
+}
+
+.DeviceListItem__title {
```
Comment (mjtognetti) [inline src/renderer/components/content-types/workspace/Device.css]: nit, have been using SUIT style naming e.g. `.DeviceListItem-title`

File: src/renderer/components/content-types/workspace/Omnibox.css [hunk 2]
```diff
@@ -12,6 +12,32 @@
   flex-direction: column;
 }
 
+.Omnibox--header {
```
Comment (mjtognetti) [inline src/renderer/components/content-types/workspace/Omnibox.css]: nit: naming convention has been `.<ComponentName>-<subcomponent>` e.g. `.Omnibox-header`

I think this was briefly discussed during the initial css conversion (though probably not firmly set in stone so could be up for discussion) but most converted CSS uses SUIT naming conventions: https://suitcss.github.io/

File: src/renderer/components/content-types/contact/ContactEditor.tsx [hunk 3]
```diff
@@ -102,11 +101,20 @@ export default function ContactEditor(props: ContentProps) {
     avatar = <img alt="avatar" src={DEFAULT_AVATAR_PATH} />
   }
 
+  let renderedDevices
+  if (devices) {
+    renderedDevices = devices.map((d) => (
+      <Content context="board" url={createDocumentLink('device', d)} />
```
Comment (mjtognetti) [inline src/renderer/components/content-types/contact/ContactEditor.tsx]: Getting a `Warning: Each child in a list should have a unique "key" prop.` message for the contact editor. Not sure if its introduced in this patch or already existed

There may be other lists in this component which need keys, but should also add a key here. Could do something like `key={d}`.

Comment (pvh) [reply src/renderer/components/content-types/contact/ContactEditor.tsx]: Ah, I'll fix that real quick.

**Pull request merged**